languagetool-language-modules/de/src/main/resources/org/languagetool/rules/de/grammar.xml

```diff
@@ -20141,6 +20174,27 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
                 <example correction="zu raten">Es mühte sie, es <marker>zuraten</marker>.</example>
             </rule>
         </rulegroup>
+        <rule id="ZUSAMMENSCHREIBUNG_ADJEKTIV_NOMEN_KOMPOSITA" name="trocken Futter oder blau Kraut in einem Wort und groß schreiben" default="temp_off">
+	        <pattern>
+	            <unify negate="yes">
+			        <feature id="case"/>
+    			    <feature id="gender"/>
+      		        <feature id="number"/>
+      		        <marker>
+                    <token regexp="yes" case_sensitive="yes">klein|groß|halb|alt|neu|blau|rot|grün|gelb|braun|grau|weiß|schwarz|schwarzweiß|bunt|bitter|
+                    süß|sauer|mürbe|mager|frisch|trocken|nass|mehrfach|best|höchst|tiefst|kleinst|mindest|falsch|exklusiv|
```

Comment:
Bitte teste hier mal, ob das mit "süß" so geht, oder ob das alles in einer Zeile stehen muss.

Reply:
Alles klar, mache ich. Irgendwas scheint da ohnehin noch nicht ganz geklappt zu haben, denn ich habe eine Fehlermeldung zum Dependency Review bekommen.

Reply:
und alle Wörter mit `ß` sollten stattdessen die Regex `(ss|ß)` bekommen